src/features/shared-link-settings-modal/__tests__/VanityNameSection-test.js

```diff
@@ -10,13 +10,15 @@ describe('features/shared-link-settings-modla/VanityNameSection', () => {
     const serverURL = 'url/';
 
     const getWrapper = (props = {}) =>
-        shallow(
+        mount(
```

Comment:
necessary?

Reply:
`Checkbox` has a subsection prop that eventually renders the component structure. Used `mount` so that the existing tests would work as before, and to expedite the change (save potential refactoring to snapshots or using `instance()` for later)